The Tabs Widget (#2356)
umbraco/UmbracoDocs: Add-ons/Starter-Kit-TEMP/Widgets/index.md

![movePage.jpg](images/The-Tabs-Widget.png)

> The Tabs Widget allows you to create tabs like the ones you find in your browser, these tabs can contain any Widget you desire.

Reviewer: ```suggestion
The Tabs Widget allows you to create tabs like the ones you find in your browser. Each tabs can contain any widgets you desire.
```

> This allows you to create multiple Widgets within one Widget this means you could have for example: a ***Text Widget***, an ***Image Widget*** and a ***Forms Widget*** all collected in one.

Reviewer: ```suggestion
You can even add multiple widgets within one tab which means you could have for example a ***Text Widget***, an ***Image Widget*** and a ***Forms Widget*** all collected in one.
```

> Where the ***Text Widget*** and the ***Image Widget*** is tab 1 and ***Forms Widget** is tab2. 

Reviewer: ```suggestion
Where the ***Text Widget*** and the ***Image Widget*** is tab 1 and ***Forms Widget*** is tab2. 
```

Reviewer: I'm a bit confused about this. Does it belong to the section above, where you say it can all be in one, or is this a different setup?

In either case, could you make that more clear some how? 🙂

Author: yea what i meant here is the tabs inside the one widget

> Below you can se the aboved mentioned example, please note the tab 1 and tab 2 is diffrently colored this was done on purpose to showcase how it could look if you choose two different theme collors.

Reviewer: ```suggestion
Below is two examples of how tabs can be set up.
```

![movePage.jpg](images/The-Tabs-Widget.png)

The Tabs Widget allows you to create tabs like the ones you find in your browser. Each tab can contain any widgets you desire.

> You can even add multiple widgets within one ***Tabs Widget*** which means you could have the ***Text Widget*** and the ***Image Widget*** on tab 1 and ***Forms Widget*** is tab2, because you can have Multiple tabs in one ***Tabs widget***. 

Reviewer: Removed the last bit, as you already mention that in the beginning of the sentence.

```suggestion
You can even add multiple widgets within one ***Tabs Widget***. This means you could have the ***Text Widget*** and the ***Image Widget*** on one tab and a ***Forms Widget*** on another tab.
```
